components/SearchBar.tsx

```diff
@@ -30,7 +30,7 @@ export const SearchBar = <T,>({
       baseData.filter((datum) =>
         searchFields.some((field) =>
           (datum[field as keyof T] as unknown as string)
-            .toLowerCase()
+            ?.toLowerCase()
```

Comment:
For some reason not all services in the Airtable have a description so this threw an error without optional chaining